### DISCUSSION TITLE
Bug: GitHub seems to think 2FA is off, even though it is enabled.

### DISCUSSION THREAD

**Discussion by Zsar** [community/community#63945] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Bug: GitHub seems to think 2FA is off, even though it is enabled: I **do** receive "Verification code"s and I **cannot** login without them.

Steps to reproduce:

	=> I received an e-mail "[ACTION REQUIRED] Your GitHub account, Zsar, will soon require 2FA" (expected: no e-mail, 2FA is enabled)
	1. I navigate to https://github.com/ , login with username and password (first factor)
	=> I am routed to a page asking me for a "Verification code" sent to me by e-mail (second factor)
	2. I log into my e-mail account, retrieve the Verification code, enter it into the page from 1. and send
	=> I am now logged in (ok)
	=> There is a large yellow banner asking me to enable 2FA (expected: no banner, 2FA is enabled)
	3. I navigate to https://github.com/settings/security
	=> There is a large text "Two-factor authentication is not enabled yet." (expected: different text, 2FA is enabled)
	4. I click on "Enable two-factor authentication" hoping this will fix the inconsistency
	=> two options available:
	- Setup authenticator app
	- SMS authentication
	(ok - e-mail is already enabled, cannot be "more" enabled; but means I cannot dis- or re-enable it to maybe overwrite the inconsistency)

It seems I cannot continue at this point, the issue has to be resolved internally.

It would seem that I can no longer report or at least find means to contact the GitHub support - whenever I follow the links to "Contact us" (e.g. [from here](https://support.github.com/features/authentication)) I only ever end up at a "Virtual Assistant" ([all pathes seem to lead here](https://support.github.com/contact/account?contact%5Btype%5D=account&flow=2fa&tags=hubberfy_authentication)), which has a dropdown menu of topics irrelevant. Given the "Topic Area" 'Bug' exists, I hope I am not completely in the wrong place. If so, would a moderator kindly move or forward this to the approprate place?

**Comment by ghost** [comment]
On the yellow banner they have a link at the 'now required' text to a Github blog: 
https://github.blog/2023-03-09-raising-the-bar-for-software-security-github-2fa-begins-march-13/

According to Github email only 2FA is not enough, they want to force you to use 2 forms of auth. SMS, TOTP Apps, and/or Security Keys (physical)

Yes this is excatly as dumb as it sounds, it's a 'feature' not a bug.

From the blog regarding emails even:
"Unlink your email in case of 2FA lockout. Since accounts on GitHub are required to have a unique email address, locked out users have difficulty starting a new account using their preferred email address—the one all their commits point to. With this feature, you can now [unlink your email address](https://github.blog/changelog/2023-02-21-unlink-your-email-in-case-of-2fa-lockout/) from a two-factor enabled GitHub account in case you’re unable to sign in or recover it. If you’re unable to find an SSH key, PAT, or a device that’s been previously signed into GitHub to recover your account, it’s easy to start fresh with a new GitHub.com account and keep that contribution graph rightfully green."

That last line is the most condescending bull I can even imagine.
